docs/examples/audio_decoder.ipynb

```diff
@@ -0,0 +1,251 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Audio Decoder in DALI\n",
+    "\n",
+    "This tutorial presents, how to set up simple pipeline, that loads and decodes audio data in DALI. Here we will use simple example from Speech Commands Data Set. While this dataset consists of samples in .wav format, following procudure can be used for most of the well-known formats for audio decoding.\n",
+    "\n",
+    "## Step-by-step guide\n",
+    "1. Let's start by importing handluf of utils and DALI itself."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from nvidia.dali.pipeline import Pipeline\n",
+    "import nvidia.dali.ops as ops            \n",
+    "import nvidia.dali.types as types\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "batch_size = 1\n",
+    "audio_files = \"audio\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "used `batch_size` is `1`, to keep things simple. You can always extend it on your own.\n",
+    "\n",
+    "2. Next, let's implement the pipeline. As you can see, it's fairly simple. Firstly, you need to load data from disk (or any other place). Used FileReader is able to load data, as well as it's labels. For more information, refer to FileReader docs. Furthermore, similarly to image data, you can use Reader ops specific for given dataset or dataset format (see COCOReader, LMDBReader)\n",
+    "\n",
+    "   After loading input, pipeline decodes the AudioData. As stated above, the AudioDecoder operator is able to decode most on well-known audio formats. Currently we support decoding operation only for the CPU.\n",
```

Comment:
> Currently we support decoding operation only for the CPU.

I bet if we start support for GPU it won't be removed. So we can skip it here. Operator description in docs will tell the user if GPU is supported or not

Reply:
Done